avatar/avatar.css

```diff
@@ -0,0 +1,156 @@
+body {
+  font-family: var(--nunito);
+  display: flex;
+  flex-direction: column;
+}
+
+.showcase-div {
+  padding: var(--spacing-lg);
+  max-width: max-content;
+  display: flex;
+  align-items: center;
+  flex-shrink: 0;
+  flex-grow: 0;
+  flex-wrap: wrap;
+  gap: var(--spacing-md);
+  justify-content: space-around;
+}
+
+.curved-avatar {
+  border-radius: 1.5rem;
+  position: relative;
+}
+
+.circle-avatar {
+  border-radius: 50%;
+  position: relative;
+}
+
+.avatar-div {
+  position: relative;
+}
+
+.avatar-xl {
+  height: 12rem;
+  width: 12rem;
+}
+
+.avatar-l {
+  height: 9rem;
+  width: 9rem;
+}
+
+.avatar-m {
+  height: 7rem;
+  width: 7rem;
+}
+
+.avatar-s {
+  height: 4rem;
+  width: 4rem;
+}
+
+.num-badge {
+  display: inline-block;
+  height: 1rem;
+  width: 1rem;
+  line-height: 1rem;
+  position: absolute;
+  background-color: var(--blue-pr);
+  border-radius: 50%;
+  font-weight: bold;
+  color: white;
+  text-align: center;
+}
+
+/* gives position to curved avatar badges */
+.badge-posi-cu {
+  top: 72%;
+  left: 90%;
+}
+
+/* gives position to cicle avatar badges */
+.badge-posi-ci {
+  top: 65%;
+  left: 85%;
+}
+
+.badge-posi-s {
+  top: 60%;
+  left: 80%;
+}
+.badge-xl {
+  padding: 0.5rem;
+}
+
+.badge-l {
+  padding: 0.4rem;
+  font-size: 0.8rem;
+}
+
+.badge-m {
+  padding: 0.2rem;
+  font-size: 0.7rem;
+}
+
+.badge-s {
+  padding: 0.04rem;
+  font-size: 0.6rem;
+}
+
+.online-badge {
+  display: inline-block;
+  padding: 0.7rem;
+  border-radius: 50%;
+  background-color: var(--green-wp);
+  position: absolute;
+}
+
+.ob-xl {
+  padding: 0.5rem;
+}
+
+.ob-l {
+  padding: 0.4rem;
+}
+
+.ob-m {
+  padding: 0.35rem;
+}
+
+.ob-s {
+  padding: 0.3rem;
+}
+
+.ob-posi-xl {
+  bottom: 1%;
+  right: 10%;
+}
+
+.ob-posi-l {
+  bottom: 3%;
+  right: 10%;
+}
+
+.ob-posi-m {
+  bottom: 4%;
+  right: 10%;
+}
+
+.ob-posi-s {
+  bottom: 7%;
+  right: 10%;
+}
+
+.ob-posi-ci {
+  bottom: 12%;
+  right: 10%;
```

Comment:
class name can be improved. if l is large lg can be used likewise sm,md,xm.
ob might not be be meaningful to user.